doc/source/user_guide/dsintro.rst

```diff
@@ -663,31 +663,6 @@ row-wise. For example:
 
    df - df.iloc[0]
 
-In the special case of working with time series data, if the DataFrame index
```

Comment:
rather than delete this, would it be appropriate to keep the example but use `df.sub(df['A'], axis=0)` to show an example of column-wise arithmetic. or is this covered elsewhere?

Reply:
The line just below this points the <basics.binop> docs which i think do this pretty well